### DISCUSSION TITLE
Is it possible to promote a deployment with the REST API?

### DISCUSSION THREAD

**Discussion by alexbjorlig** [vercel/community#3474] [Feature Question] [answered]
### Question

Our use-case is to deploy our application, on a fixed schedule every night.

I was thinking about doing this with the "promote" feature.

But I can't find anything about this with the REST API?

Is it only possible using the CLI?

**Comment by amyegan** [comment, marked as answer]
Hey @alexbjorlig. You can use the "[create a new deployment](https://vercel.com/docs/rest-api/endpoints#create-a-new-deployment)" endpoint to redeploy an existing preview deployment into production. Something like this should be all you need.
```
curl --request POST \
  --url https://api.vercel.com/v13/deployments \
  --header 'Authorization: Bearer <TOKEN>' \
  --data '{
    "deploymentId": "dpl_FjnsYTCxghagj4F5qvk4wxhfybJ4",
    "target": "production",
    "name": "my-test-app"
  }'
```

**Reply by alexbjorlig** [reply]
Will give this a try - thanks 👍
